styles/mixins/patterns/_molecules.scss

```diff
@@ -59,3 +59,33 @@ $-space: var.$layout-space;
     @include layout.vertical-rhythm($-space * 0.5);
   }
 }
+
+// A responsive sized and positioned container for modal content
+@mixin modal {
+  // The modal container is positioned horizontally- and vertically-centered
+  @include layout.fixed-centered;
+
+  // For narrower viewports/default:
+  // - positioning: centered vertically and horizontally in the viewport
+  // - sizing: fills up most of the available horizontal space
+  width: 90vw;
+  max-width: 90vw;
+  max-height: 90vh;
+
+  // For wider viewports:
+  // - positioning: keep horizontal centering but bring top nearer to top
+  //     of viewport.
+  // - sizing: Set a min-width to make sure the modal isn't too dinky. Set
+  //     a max-width to make sure it isn't too ginormously wide on wide screens.
+  @media screen and (min-width: 48rem) {
+    top: 10vh;
+    transform: translate(-50%, 0);
+
+    width: auto;
+    min-width: 28rem;
+    max-width: 44rem; // default for older browsers
+    max-width: min(44rem, 90vw);
```

Comment:
Makes sense to have a fallback. From https://developer.mozilla.org/en-US/docs/Web/CSS/min()#browser_compatibility it looks like Safari has supported `min` for a long time but support in Chrome and Firefox is much more recent. That's unusual.